### ISSUE TITLE
Remove non npm dependency

### ISSUE THREAD

**Issue by feross** [royriojas/esformatter-jsx#40]
This package is depending on the tarball hosted here:

```
https://github.com/royriojas/fresh-falafel/tarball/fix-parse-issue
```

This is currently breaking offline installs using the local npm cache. Is there anyway not to depend on this?

(re-posted from  https://github.com/feross/standard/issues/326)

**Comment by feross** [comment]
cc @mafintosh

**Comment by maxogden** [comment]
@substack if you merge https://github.com/substack/node-falafel/pull/51 it might help

**Comment by royriojas** [comment]
For now I published that branch as `fresh-falafel` until @substack has time to review/merge  substack/node-falafel#51

**Comment by royriojas** [comment]
published `esformatter-jsx@2.3.11` that doesn't use dependencies from github

**Comment by feross** [comment]
Nice, thanks!
